features/admin.server-configurations.v1/models/index.ts

```diff
@@ -19,3 +19,4 @@
 export * from "./endpoints";
 export * from "./governance-connectors";
 export * from "./password-recovery";
+export * from "./username-recovery";
```

Comment:
Let's not update the index file. instead take the models from the corresponding file

Reply:
Fixed by https://github.com/wso2/identity-apps/pull/7004/commits/59d040047dd8b21efd511e781c167e8ebd65b64f